tests/test_assets/model_details/input/model_details_model_results_info_good_test_input.csv

```diff
@@ -0,0 +1,20 @@
+name,gene_expression,disease_correlation,pathology,biomarkers
+3xTg-AD,TRUE,,TRUE,TRUE

```

Comment:
Can you add another test input file to check the following cases:
```
gene_expression,disease_correlation
3xTg-AD,,TRUE, ...
3xTg-AD,TRUE,TRUE, ...
```
So far I'm only seeing output for `gene_expression` = TRUE.

If the test is only being run on one set of data (3xTg-AD), then you also don't need the rest of the models this file. Or you can just add one extra model to the file to show that it doesn't use non-relevant lines.